README.md

```diff
@@ -1356,6 +1356,57 @@ $ ssh-add -E md5 -l
 
 When using the key `pinentry` will be invoked to request the key's passphrase. The passphrase will be cached for up to 10 minutes idle time between uses, to a maximum of 2 hours.
 
+## Remote Machines (agent forwarding)
+
+If you want to use your YubiKey to sign a git commit on a remote machine, or ssh through another layer, then this is possible using "Agent Forwarding".  To do this, you need to already have shell access to your remote machime, and your YubiKey setup on your host machine.
+
+- First, on your host machine run:
```

Comment:
I know "host" almost always means a remote rather than local machine, but I'd still prefer the language "remote host" just to be super clear which is which.